runtime/common/Cargo.toml

```diff
@@ -144,3 +144,6 @@ try-runtime = [
   "pallet-evm-chain-id/try-runtime",
   "fp-self-contained/try-runtime",
 ]
+
+# Set timing constants (e.g. session period) to faster versions to speed up testing.
+fast-runtime = []
```

Comment:
Not sure TBH, when `fast-runtime` and when `testnet-runtime` is used. Seems like this should be under `fast-runtime`. I understand real production nodes will not use `fast-runtime`. 

Why both features are not combined into just one?

Reply:
> Why both features are not combined into just one?

I think the main reason is lazyness from our side as the `fast-runtime` is "inherited" from Substrate. It comes with macro support, i.e. `prod_or_fast!` for setting parameter types. 

IMO, we should merge both features. Additionally, we have been building `fast-runtime` Altair WASMs over the past months even though there is NO use for them, we only need it for Centrifuge runtime as our Catalyst test network runs with it. 

So most ideally, we should only have a `testnet-runtime` feature such that Altair@testnet-runtime is the Algol WASM and Centrigue@testnet-runtime is the Catalyst WASM.

Reply:
Ok, I agree with you! Thanks @wischli for all your explanations regarding this topic